### PR TITLE
Add 2.1 features to .babylon file loader

### PR DESCRIPTION
**Pull request by Palmer-JC**
Add Mesh.FreezeWorldMatrix(), Material.checkReadyOnlyOnce, & Camera.setCameraRigModeSetting camera rig to RIG_MODE_VR likely has issues, which will be addressed later.

### CODE REVIEW

**Comment by Palmer-JC** [comment]
Also, Texture.Base64String for inline texture files

File: src/Loading/Plugins/babylon.babylonFileLoader.ts [hunk 1]
```diff
@@ -48,7 +47,11 @@
             texture = new BABYLON.RenderTargetTexture(parsedTexture.name, parsedTexture.renderTargetSize, scene);
             texture._waitingRenderList = parsedTexture.renderList;
         } else {
-            texture = new BABYLON.Texture(rootUrl + parsedTexture.name, scene);
+            if (parsedTexture.Base64String){
```
Comment (deltakosh) [inline src/Loading/Plugins/babylon.babylonFileLoader.ts]: Can you please use base64String instead of Base64String

**Comment by deltakosh** [comment]
I updated the file format spec:
http://doc.babylonjs.com/page.php?p=22661

I integrate this PR and I will change the base64String naming for you

**Comment by Palmer-JC** [comment]
Thanks.  If doc site source really going to github, not worth it getting my docsite account working.  Did proof though.  checkReadyOnlyOnce should also be listed as optional, right?

Getting this in makes it much easier for me, especially since exporter PR from jackcaron.  Had already transferred most of my first pass changes to .babylon exporter.  Need to set that file aside, do another pull to get those.

Netbeans is really good at diffing & side-by-side editing of 2 unrelated files.  Will be easy to move my changes to the latest PR'ed version.  That is also how I move changes back & forth between the exporters.

**Comment by deltakosh** [comment]
I will mark it as optional

**Pull request merged**